constructor/fcp.py

```diff
@@ -8,17 +8,19 @@
 """
 from __future__ import absolute_import, division, print_function
 
-from collections import defaultdict
+from collections import defaultdict, Counter
```

Comment:
Oh `Counter` is unused now

```suggestion
from collections import defaultdict
```